src/Sulu/Bundle/DocumentManagerBundle/Resources/translations/sulu/backend.de.xlf

```diff
@@ -14,21 +14,13 @@
                 <source>sulu-document-manager.publish</source>
                 <target>Veröffentlichen</target>
             </trans-unit>
-            <trans-unit id="4" resname="sulu-document-manager.published-shadow">
-                <source>sulu-document-manager.published-shadow</source>
-                <target>Der Veröffentlichungsstatus wird vom Status der Hauptseite übernommen.</target>
-            </trans-unit>
```

Comment:
@alexander-schranz I think you moved these translations to this file. They don't belong here, because they are already referring to pages, which would be wrong to use in other contexts. Please introduce other translation keys where you need them.